File: LondonMeetup/app/src/main/java/com/suyashsrijan/londonmeetup/API/tfl/enums/Modes.java
package com.suyashsrijan.londonmeetup.API.tfl.enums;

public enum Modes {
    BUS {
        public String toString() {
            return "bus";
        }
    },

    TUBE {
        public String toString() {
            return "tube";
        }
    },

    CYCLE {
        public String toString() {
            return "cycle";
        }
    }
}

File: LondonMeetup_v2/app/src/main/java/com/suyashsrijan/londonmeetup/oldcode/API/tfl/enums/Modes.kt
package com.suyashsrijan.londonmeetup.API.tfl.enums

enum class Modes {
    BUS {
        override fun toString(): String {
            return "bus"
        }
    },

    TUBE {
        override fun toString(): String {
            return "tube"
        }
    },

    CYCLE {
        override fun toString(): String {
            return "cycle"
        }
    }
}
